### PR TITLE
Mp/fix issues

### LINKED ISSUE #53: Update React State for rendering components as opposed to relying on display: none

### LINKED ISSUE #57: Wrap all the pages inside of navbar and footer

### LINKED ISSUE #59: Finalize the font options

### LINKED ISSUE #60: Change the name of the Interfaces.tsx file and move it into @types folder.

### LINKED ISSUE #61: Fix the position of sorting buttons to always be visible and remove/redesign the scroll bar.

### PR DESCRIPTION
**Pull request by milos381**
### Summary

Fixed issues 53 50 57 59 60 61

### Type of Change

Please delete options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)

### Issues

closes #53
#50 
closes #57 
closes #59 
closes #60 
closes #61 

What is left from the existing issues is responsive design, which I will work on after I finish the algorithm, if that's ok, since there will probably still be some stuff that was added on the frontend.

### CODE REVIEW

File: client/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                        ? '#092173'
+                        : pathname === '/login'
+                            ? '#092173'
+                            : pathname === '/dashboard'
+                                ? '#092173'
+                                : '#fff',
+            }}
+        >
+            <div>
+                <h4>Copyright © Team Cat-Snake 2022</h4>
```
Comment (evanmcneely) [inline client/components/Footer.tsx:33]: Maybe change "Team Cat-Snake" to "GraphQL-Gateway"? Thoughts?

**Comment by evanmcneely** [comment]
Issue #50 was meant to imply making the github button more prominent and more likely to be used like below. Just suggestions

Screen Shot 2022-07-08 at 5 48 55 PM

**Comment by evanmcneely** [comment]
Running the app I'm experiencing a new glitch that wasn't there before.

Screen Shot 2022-07-08 at 5 46 01 PM

**Comment by evanmcneely** [comment]
All in all good job, I like the graphics and navigation between login/signup/landing page in smooth.

**Review by donjewey — APPROVED** [review]

**Review by evanmcneely — APPROVED** [review]

**Comment by milos381** [comment]
:)

**Pull request merged**